catalog.html

```diff
@@ -56,23 +56,201 @@
+                        <div class="faq__answer">
+                        </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Background</summary>
+                        <div class="faq__answer">
+                        </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Special</summary>
+                    <div class="faq__answer">
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="yellow" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Yellow</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="green" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Green</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="mix" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Mix</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="orange" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Orange</span>
+                            </span>
+                        </label>
+                    </div>
+                </details>
+                <button type="submit" class="btn btn--color btn--full">Apply</button>
+            </form>
         </aside>
 
         <article class="blog-content">
 
             <section class="catalog catalog--big">
+                <div class="catalog-filters">
+                    <h2 class="subtitle subtitle--left">All item</h2>
+                    <div class="catalog-controls">
+
+                        <form class="form-select" action="#" method="get">
+                            <select class="select" name="sort" id="sort">
```

Comment:
Structure:
```
<form>
  <label>
    <select>.
      <option>..</option>
    </select>
   ::after
  </label>
</form>
```
Please style select (add font, color, border, paddings) 
Also we can customise the arrow (appearence: none + after) 
And we style drop down a little bit (styles for option)

Reply:
done